pkg/network/gvtap/gvtap.go

```diff
@@ -0,0 +1,492 @@
+package gvtap
```

Comment:
bikeshedding: this should be called `gvisortapvsock` to match the driver name?

Reply:
There seems to be another thing called "gvtap"
https://docs.gigamon.com/doclib511/Content/GV-Cloud-Any/AnyCloud_Install_G-vTAP_Agents.html

Reply:
fixed